<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="QA1072" name="com.pos.testcases.EditVendor" tests="2" failures="0" timestamp="2023-09-29T20:07:56 IST" time="114.372" errors="2">
  <testcase name="runLogin" time="98.388" classname="com.pos.testcases.EditVendor">
    <error type="java.lang.RuntimeException" message="See the reporter for details.">
      <![CDATA[java.lang.RuntimeException: See the reporter for details.
at com.framework.utils.Reporter.reportStep(Reporter.java:101)
at com.framework.utils.Reporter.reportStep(Reporter.java:116)
at com.framework.selenium.api.base.SeleniumBase.locateElement(SeleniumBase.java:657)
at com.pos.pages.LoginPage.enterUserName(LoginPage.java:9)
at com.pos.pages.LoginPage.login(LoginPage.java:27)
at com.pos.testcases.EditVendor.runLogin(EditVendor.java:24)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
    </error>
  </testcase> <!-- runLogin -->
  <system-out/>
  <testcase name="runLogin" time="15.984" classname="com.pos.testcases.EditVendor">
    <error type="java.lang.RuntimeException" message="See the reporter for details.">
      <![CDATA[java.lang.RuntimeException: See the reporter for details.
at com.framework.utils.Reporter.reportStep(Reporter.java:101)
at com.framework.utils.Reporter.reportStep(Reporter.java:116)
at com.framework.selenium.api.base.SeleniumBase.locateElement(SeleniumBase.java:657)
at com.pos.pages.LoginPage.enterUserName(LoginPage.java:9)
at com.pos.pages.LoginPage.login(LoginPage.java:27)
at com.pos.testcases.EditVendor.runLogin(EditVendor.java:24)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
    </error>
  </testcase> <!-- runLogin -->
  <system-out/>
</testsuite> <!-- com.pos.testcases.EditVendor -->
